config/crd/bases/sw.kogito.kie.org_kogitoserverlessworkflows.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     - jsonPath: .status.address.url
       name: Address
       type: string
-    - jsonPath: .status.condition
+    - jsonPath: .status.conditions[?(@.type=='Running')].status
```

Comment:
Could we instead give the latest condition type ?
Or call it `RUNNING` instead of Condition ?

Also, I thin the `REASON` needs to be changed
```
$ kget KogitoServerlessWorkflow
NAME       PROFILE   VERSION   ADDRESS   CONDITION   REASON
greeting             0.0.1               False       
```

Reply:
> Could we instead give the latest condition type ?

What do you mean?

Actually, I need to change this. It should be:

```
$ kget KogitoServerlessWorkflow
NAME       PROFILE   VERSION   ADDRESS   READY   REASON
greeting             0.0.1               False       
```

Condition > Ready. And the reason should be changed, as you pointed! Nice catch.